.env.sample

```diff
@@ -2,4 +2,5 @@ ENVIRONMENT=local
 APP_NAME=seedling
 LOCAL_APP_PORT=8050
 LOCAL_MSSQL_PORT=1433
-LOCAL_IMAGE_NAME=flowehr_seedling_app
\ No newline at end of file
+LOCAL_IMAGE_NAME=flowehr_seedling_app
+APPLICATIONINSIGHTS_CONNECTION_STRING=application_insight_connection_string
```

Comment:
nit: please add new-line to end of the file

Reply:
Done 543c621b542bb9b30ae0612dd9f2bb1982ed2426